tests/test_filepath.py

```diff
@@ -56,14 +57,13 @@ def test_filepath_s3():
 def test_duplicate_upload(store="repo"):
     ext = schema.external[store]
     stage_path = dj.config['stores'][store]['stage']
-    filename = 'plot.dat'
     relpath = 'one/two/three'
-    os.makedirs(os.path.join(stage_path, relpath), exist_ok=True)
-    managed_file = os.path.join(stage_path, relpath, filename)
-    with open(managed_file, 'wb') as f:
+    managed_file = Path(stage_path, relpath, 'plot.dat')
+    managed_file.parent.mkdir(parents=True, exist_ok=True)
+    with managed_file.open('wb') as f:
         f.write(os.urandom(300))
-    ext.fput(managed_file)
-    ext.fput(managed_file)   # this is fine because the file is the same
+    ext.upload_filepath(managed_file)
+    ext.upload_filepath(managed_file)  # this is fine because the file is the same
 
 
 def test_duplicate_upload_s3():
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was `dj/repo/one\two\three\plot.dat`.